hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -204,8 +214,14 @@ private void summarizeAsProto(Instant exportTimeStamp, AllAccountBalances.Builde
 
     private boolean exportBalancesProtoFile(
             AllAccountBalances.Builder allAccountsBuilder, String protoLoc) {
-        try (FileOutputStream fout = new FileOutputStream(protoLoc)) {
-            allAccountsBuilder.build().writeTo(fout);
```

Comment:
Slightly preferable to precede this with,
```
        accountBalanceDigest.reset();
```
since it's least _conceivable_ the last export attempt failed after a non-zero number of `update()` calls but _before_ `digest()`.

Reply:
Ooh, indeed, thanks Michael --- added  `accountBalanceDigest.reset()` now at the beginning of the method.